### PR TITLE
Release Note for WebView2 with Stable 105 and Pre-release 106

### PR DESCRIPTION
**Pull request by imicy**
Rendered page for review:

https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/release-notes?branch=pr-en-us-2153

### CODE REVIEW

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -34,6 +34,54 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
 
 
 <!-- ====================================================================== -->
+## 1.0.1369-prerelease
+
+Release Date: Sep 4, 2022
```
Comment (mikeshhoffman) [inline microsoft-edge/webview2/release-notes.md:39]:
```suggestion
Release Date: September 4, 2022

```

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -34,6 +34,54 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
...
+   * `Drop`
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2CompositionController Class](/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller?view=webview2-dotnet-1.0.1369-prerelease&preserve-view=true)
+
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2CompositionController Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2compositioncontroller?view=webview2-winrt-1.0.1369-prerelease&preserve-view=true)
```
Comment (mikeshhoffman) [inline microsoft-edge/webview2/release-notes.md:64]:
```suggestion
* [CoreWebView2CompositionController.DragLeave Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2compositioncontroller?view=webview2-winrt-1.0.1369-prerelease&preserve-view=true#dragleave)

```
Will there be now the other 3 methods, for .NET & WinRT?  In addition to DragLeave, Win32 also has DragEnter, DragOver, and Drop:
https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true

File: microsoft-edge/webview2/release-notes.md [hunk 3]
```diff
@@ -34,6 +34,54 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
...
+
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2CompositionController3 Class](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true)
```
Comment (mikeshhoffman) [inline microsoft-edge/webview2/release-notes.md:69]:
```suggestion
* [ICoreWebView2CompositionController3.DragEnter method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true#dragenter)
* [ICoreWebView2CompositionController3.DragLeave method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true#dragleave)
* [ICoreWebView2CompositionController3.DragOver method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true#dragover)
* [ICoreWebView2CompositionController3.Drop method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true#drop)

```

File: microsoft-edge/webview2/release-notes.md [hunk 4]
```diff
@@ -34,6 +34,54 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
...
+
+*  The drag and drop API: 
+   * `DragEnter`
+   * `DragLeave`
+   * `DragOver`
+   * `Drop`
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2CompositionController Class](/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller?view=webview2-dotnet-1.0.1369-prerelease&preserve-view=true)
```
Comment (mikeshhoffman) [inline microsoft-edge/webview2/release-notes.md:59]:
```suggestion
* [CoreWebView2CompositionController.DragLeave Method](/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller.dragleave?view=webview2-dotnet-1.0.1369-prerelease#microsoft-web-webview2-core-corewebview2compositioncontroller-dragleave)

```

File: microsoft-edge/webview2/release-notes.md [hunk 5]
```diff
@@ -34,6 +34,54 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
...
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2CompositionController3 Class](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true)
+
+---
+
+#### Bug fixes
+
+*   Fix a bug where WPF apps would crash when windows with WebView2 were closed. ([Issue #640](https://github.com/MicrosoftEdge/WebView2Feedback/issues/640))
```
Comment (mikeshhoffman) [inline microsoft-edge/webview2/release-notes.md:75]:
```suggestion
*   Fixed a bug where WPF apps would crash when windows with WebView2 were closed. ([Issue #640](https://github.com/MicrosoftEdge/WebView2Feedback/issues/640))

```

File: microsoft-edge/webview2/release-notes.md [hunk 6]
```diff
@@ -34,6 +34,54 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
...
+
+*   Fixed a bug that produced simultaneous WebView creation failure. This change is Runtime-specific. ([Issue #2703](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2703))  
+
+*   Fixed print settings paper size to support dimensions as small as 0.01 inches (Runtime).  
+
+*   Fixed a bug that WebView2 print dialog may reset scale to fit to printable everytime. ([Issue #2523](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2523))
```
Comment (mikeshhoffman) [inline microsoft-edge/webview2/release-notes.md:81]:
```suggestion
*   Fixed a bug where the WebView2 print dialog reset the **Scale** setting to **Fit to printable area** every time. ([Issue #2523](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2523))  
```

File: microsoft-edge/webview2/release-notes.md [hunk 7]
```diff
@@ -34,6 +34,54 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
 
 
 <!-- ====================================================================== -->
+## 1.0.1369-prerelease
```
Comment (JasonAndrewWriter) [inline microsoft-edge/webview2/release-notes.md:37]: @imicy Will there be a release H2 section in this PR?

Comment (JasonAndrewWriter) [inline microsoft-edge/webview2/release-notes.md:37]: @nishitha-burman Is the overview for the api articles overview need to be updated.  Will it be in a seperate pr?    Will we need to update it?

Comment (nishitha-burman) [reply microsoft-edge/webview2/release-notes.md:37]: Yes, that will be in a separate PR. @imicy

Comment (JasonAndrewWriter) [reply microsoft-edge/webview2/release-notes.md:37]: Sweet.  Please add me to the pr.

Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md:37]: https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1343.22 - "a day ago"

File: microsoft-edge/webview2/release-notes.md [hunk 8]
```diff
@@ -34,6 +34,62 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
```
Comment (JasonAndrewWriter) [inline microsoft-edge/webview2/release-notes.md:34]: Rendered page for review:

https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/release-notes?branch=pr-en-us-2153

**Review by JasonAndrewWriter — APPROVED** [review]

File: microsoft-edge/webview2/release-notes.md [hunk 9]
```diff
@@ -34,6 +34,75 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
...
+
+Release Date: September 6, 2022
+
+[NuGet package for WebView2 SDK 1.0.1343.22](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1343.22)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 105.0.1343.22 or higher.
+
+### General
+
+* Fixed a bug in the wv2winrt tool where a WinMD file wasn't referenced in some projects.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:47]:
```suggestion
* Fixed a bug in the **wv2winrt** tool where a WinMD file wasn't referenced in some projects.

```

**Review by JasonAndrewWriter — APPROVED** [review]

**Review by JasonAndrewWriter — APPROVED** [review]

**Review by JasonAndrewWriter — APPROVED** [review]

File: microsoft-edge/webview2/release-notes.md [hunk 10]
```diff
@@ -34,6 +34,76 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
...
+#### Bug fixes
+
+*   Fixed a bug where WPF apps would crash when windows with WebView2 were closed. ([Issue #640](https://github.com/MicrosoftEdge/WebView2Feedback/issues/640))
+
+
+*   Fixed a bug that produced simultaneous WebView creation failure. This change is Runtime-specific. ([Issue #2703](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2703))  
+
+*   Fixed print settings paper size to support dimensions as small as 0.01 inches (Runtime).  
+
+*   Fixed a bug that WebView2 print dialog may reset scale to fit to printable everytime. ([Issue #2523](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2523))
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:103]: My rewrite got lost.  Checking past comments...

Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:103]:
```suggestion
*   Fixed a bug where the WebView2 print dialog reset the **Scale** setting to **Fit to printable area** every time. ([Issue #2523](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2523))

```

**Review by JasonAndrewWriter — APPROVED** [review]

**Review by JasonAndrewWriter — APPROVED** [review]

File: microsoft-edge/webview2/release-notes.md [hunk 11]
```diff
@@ -34,6 +34,75 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
...
 <!-- ====================================================================== -->
+## 1.0.1343.22
+
+Release Date: September 6, 2022
+
+[NuGet package for WebView2 SDK 1.0.1343.22](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1343.22)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 105.0.1343.22 or higher.
+
+### General
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:45]:
```suggestion
### General

No changes in this release.

```

File: microsoft-edge/webview2/release-notes.md [hunk 12]
```diff
@@ -34,6 +34,75 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
...
+
+Release Date: September 6, 2022
+
+[NuGet package for WebView2 SDK 1.0.1369-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1369-prerelease)
+
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 106.0.1369.0 or higher. 
+
+### General
+
+* Fixed a bug that produced simultaneous WebView creation failure (Runtime) [Issue #2703](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2703)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:61]:
```suggestion
* Fixed a bug that produced simultaneous WebView creation failure (Runtime). [Issue #2703](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2703)

```

File: microsoft-edge/webview2/release-notes.md [hunk 13]
```diff
@@ -34,6 +34,79 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
...
+* [ICoreWebView2CompositionController3.Drop method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1369-prerelease&preserve-view=true#drop)
+
+
+---
+
+#### Bug fixes
+
+*   Fixed a bug where WPF apps would crash when windows with WebView2 were closed. ([Issue #640](https://github.com/MicrosoftEdge/WebView2Feedback/issues/640))
+
+*   Fixed a bug that produced simultaneous WebView creation failure (Runtime) [Issue #2703](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2703)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:100]:
```suggestion
*   Fixed a bug that produced simultaneous WebView creation failure (Runtime). [Issue #2703](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2703)

```

File: microsoft-edge/webview2/release-notes.md (hunk 10 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md]: @JasonAndrewWriter  - this change was lost, i will re-enter 3rd time.  trying to change from:

* Fixed a bug that WebView2 print dialog may reset scale to fit to printable everytime. ([Issue #2523](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2523))

to:

* Fixed a bug where the WebView2 print dialog reset the **Scale** setting to **Fit to printable area** every time. ([Issue #2523](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2523))

File: microsoft-edge/webview2/release-notes.md [hunk 14]
```diff
@@ -34,6 +34,80 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
...
+#### Bug fixes
+
+*   Fixed a bug where WPF apps would crash when windows with WebView2 were closed. ([Issue #640](https://github.com/MicrosoftEdge/WebView2Feedback/issues/640))
+
+*   Fixed a bug that produced simultaneous WebView creation failure (Runtime). [Issue #2703](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2703)
+
+
+*   Fixed print settings paper size to support dimensions as small as 0.01 inches (Runtime).  
+
+*   Fixed a bug that WebView2 print dialog may reset scale to fit to printable everytime. ([Issue #2523](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2523))
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:105]:
```suggestion
* Fixed a bug where the WebView2 print dialog reset the **Scale** setting to **Fit to printable area** every time. ([Issue #2523](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2523))
```
3rd entry of this Suggestion

**Review by JasonAndrewWriter — APPROVED** [review]

**Review by mikehoffms — APPROVED** [review]

**Review by JasonAndrewWriter — APPROVED** [review]

**Review by JasonAndrewWriter — APPROVED** [review]

**Comment by captainbrosset** [comment]
Code is now in main and live.

**Pull request merged**